src/xFactor/Main.java

```diff
@@ -0,0 +1,145 @@
+package xFactor;
+
+import java.util.Scanner;
+import java.util.Set;
+import java.util.concurrent.TimeUnit;
+
+public class Main {	
+	public static void main(String[] args) {	
+		Scanner scan = new Scanner(System.in);
+		System.out.println("Hi there!" + "\n" + "This is X-Factor!" + "\n" + "Please choose one of the options below:" + "\n"
+				+ "1 - 4 Judges, 15 Participants, 5 Stages" + "\n"
+				+ "2 - 3 Judges, 8 Participants, 3 Stages"  + "\n"
+				+ "To exit, press 0.");
+		
+		int input = scan.nextInt();
+		scan.close();
+		
+		switch(input) {
+			case 1: {
+				Application app = new Application();

```

Comment:
Extract the creation logic in different services, not only in different methods. Classes are like wrappers for functionality, they should also be responsible for single thing